### ISSUE TITLE
[issue]: Devuan 5 not getting past ISOLINUX boot menu, mount error.

### ISSUE THREAD

**Issue by it9exm** [ventoy/Ventoy#2696]
### Official FAQ

- [X] I have checked the official FAQ.

### Ventoy Version

1.0.96

### What about latest release

Yes. I have tried the latest release, but the bug still exist.

### Try alternative boot mode

Yes. I have tried them, but the bug still exist.

### BIOS Mode

Both

### Partition Style

GPT

### Disk Capacity

8 GB

### Image file checksum (if applicable)

Yes.

### Image file download link (if applicable)

https://mirror.leaseweb.com/devuan/devuan_daedalus/installer-iso/devuan_daedalus_5.0.1_i386_netinstall.iso

### What happened?

Hello, I tried to start Devuan 5 (which is Debian without Systemd) using Ventoy but I get the following error:

```
mount: mounting LABEL=DEVUAN501 on /cdrom failed: No such file or directory
mount: mounting UUID= on /cdrom failed: No such file or directory

```
The same issue has been reported in the forums at the following link 

https://forums.ventoy.net/showthread.php?tid=2722&pid=7960

and also happens to @asbestomolesto in the following video around 12.24 time marker, he is using an older version, please forgive him for cursing at Ventoy!

https://www.youtube.com/watch?v=NmyjRDfXhkA

I'm not aware of the inner workings of Ventoy so I don't know whether it is causing the issue but I think maybe it just needs the proper hook to be added?

**Comment by steve6375** [comment]
If you use Easy2Boot, you can run the shell and type
mount /dev/sda4 /cdrom
to mount the E2B partition 4 (which is the ISO) as /cdrom
then type 'exit' and continue install

tested using Legacy mode.

**Comment by asbestomolesto** [comment]
Yes, please apologize for my cursing in the video: my videos are comical, full of irony and bad jokes so please don't take me seriously. They depict a generic frustrated guy unable to do basic stuff (that's me! hahahahah!) 

But the bug / problem is there... 

PS also TempleOS isn't booting!!

**Comment by dashfader** [comment]
Will there ever be solution to this? It's still happening on Ventoy 1.0.99
With the same error, trying to mount /dev/sdb1 manually (Ventoy partition) also doesn't work

```
mount: mounting LABEL=DEVUAN501 on /cdrom failed: No such file or directory
mount: mounting UUID= on /cdrom failed: No such file or directory
```

**Comment by steve6375** [comment]
Did you try VTOY_LINUX_REMOUNT
https://ventoy.net/en/plugin_control.html

**Comment by dashfader** [comment]
> Did you try VTOY_LINUX_REMOUNT https://ventoy.net/en/plugin_control.html

I did, and it doesn't help anything. Still got the same error
Trying to boot through Ventoy GRUB2 Mode only throws it back to ventoy main menu

And trying to boot through memdisk mode only stuck at copy imagedisk to ram forever

**Comment by steve6375** [comment]
You can add e2b to a Ventoy USB stick for legacy booting of devuan
https://easy2boot.xyz/ventoy/add-easy2boot-to-a-ventoy-usb-drive/

**Comment by bobemoe** [comment]
What needs to be done to support Devuan 5 on Ventoy? 

Everywhere seems to suggest easy2boot. But I would rather help add support to Ventoy than jump ship...?
